pkg/scaling/scale_handler.go

```diff
@@ -464,6 +464,8 @@ func buildScaler(triggerType string, config *scalers.ScalerConfig) (scalers.Scal
 		return scalers.NewRedisStreamsScaler(config)
 	case "stan":
 		return scalers.NewStanScaler(config)
+	case "swift":
```

Comment:
```suggestion
	case "openstack-swift":
```

Reply:
Constructor and other helper functions were named `NewSwiftScaler` or `parseSwiftMetadata`. Would it be the case to rename them to `NewOpenStackSwiftScaler` and `parseOpenStackSwiftMetadata`, for example?

Reply:
yeah

Reply:
Both 0313be5 and e12e140 applied the suggested changes 😃